### ISSUE TITLE
Crash on load with CircuitControlledSilo mod

### ISSUE THREAD

**Issue by dbeckwith** [raiguard/Factorio-RateCalculator#12]
#### Describe the Bug

I've had this mod installed for a while and just installed [`CircuitControlledSilo`](https://mods.factorio.com/mod/CircuitControlledSilo) and now get this error on startup:

```
Failed to load mods: Error in assignID: entity with name 'smart-rocket-silo-red' does not exist. It was removed by CircuitControlledSilo.

Source: rcalc-selection-tool (selection-tool).

Mods to be disabled:
• CircuitControlledSilo (1.0.2)
• RateCalculator (1.0.3)
• extended-descriptions (1.0.12)
```

`extended-descriptions` is probably not involved, it just touches everything.

#### To Reproduce

1. Load the save file

#### Save file & Username

[Save file](https://www.dropbox.com/s/r80bov5ko5huhfv/save.tar.gz?dl=0)

My username is `sonaxaton`

#### Log file

[factorio-current.log](https://github.com/raiguard/Factorio-RateCalculator/files/5213117/factorio-current.log)

**Comment by dbeckwith** [comment]
Just FYI I'm no longer using CircuitControlledSilo (I realized it's kind of pointless since I can just put a circuit condition on the satellite inserter).

**Comment by raiguard** [comment]
Well, I can't reproduce this on my current code, so I'm calling it fixed with 1.1.
